apps/new/widget/page/project/Index.jsx

```diff
@@ -1,8 +1,23 @@
+const { id } = props;
+const { getProjectMeta } = VM.require(
+  "${config_account}/widget/lib.projects",
+) || {
+  getProjectMeta: () => {},
+};
+
+const data = getProjectMeta(id);
+if (!id || !data) {
+  return "Loading...";
+}
+
 const { Layout } = VM.require(
   "${config_account}/widget/page.project.Layout",
 ) || {
   Layout: () => <></>,
 };
+
+const projectId = props.id.split("/")[2] ?? null;
```

Comment:
let's add it in `lib`, something like `getProjectIdFromProps`
I also used it in `tasks`, can you also update it there.
thanks

Reply:
I like this too -- `getProjectIdFromPath`